templates/books-show.html

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+ <div class="row">
+   <div class="col-md-10 col-md-offset-1">
+     <div class="row">
+       <div class="col-md-10 col-md-offset-1">
```

Comment:
why are there so many divs in divs in divs that you never use???? this reeks of copy/pasting and never looking back at your html.